build-conventions/src/main/java/org/elasticsearch/gradle/internal/conventions/LicensingPlugin.java

```diff
@@ -36,15 +37,18 @@ public void apply(Project project) {
              isSnapshotVersion(project) ? revision.get() : "v" + project.getVersion()
         );
 
-        Provider<String> projectLicenseURL = licenseCommitProvider.map(licenseCommit -> ELASTIC_LICENSE_URL_PREFIX +
+        Provider<String> elasticLicenseURL = licenseCommitProvider.map(licenseCommit -> ELASTIC_LICENSE_URL_PREFIX +
                 licenseCommit + ELASTIC_LICENSE_URL_POSTFIX);
+        Provider<String> agplLicenseURL = licenseCommitProvider.map(licenseCommit -> ELASTIC_LICENSE_URL_PREFIX +
+            licenseCommit + AGPL_ELASTIC_LICENSE_URL_POSTFIX);
         // But stick the Elastic license url in project.ext so we can get it if we need to switch to it
-        project.getExtensions().getExtraProperties().set("elasticLicenseUrl", projectLicenseURL);
+        project.getExtensions().getExtraProperties().set("elasticLicenseUrl", elasticLicenseURL);
 
         MapProperty<String, String> licensesProperty = project.getObjects().mapProperty(String.class, String.class).convention(
                 providerFactory.provider(() -> Map.of(
                         "Server Side Public License, v 1", "https://www.mongodb.com/licensing/server-side-public-license",
-                        "Elastic License 2.0", projectLicenseURL.get())
+                        "Elastic License 2.0", elasticLicenseURL.get(),
+                        "The OSI-approved Open Source license Version 3.0", agplLicenseURL.get())
```

Comment:
Is this the correct name? Should this be the "Gnu Affero General Public LIcense Version 3"?

Reply:
That name is what is checked against by the release manager: https://github.com/elastic/infra/blob/a1eb57745bd557119d03979363147c50e9cd4c5d/cd/release/release-manager/stack-configurations-plugin/src/main/groovy/org/elastic/gradle/StackArtifact.groovy#L51

Reply:
That seems... odd. "OSI-approved Open Source license Version 3.0" isn't a thing. I think that text should be updated in the release manager. cc @navyau09